src/server.go

```diff
@@ -124,7 +155,7 @@ func (a *App) ServeHTTP(w http.ResponseWriter, r *http.Request) {
       good := false
       for _, vn := range rand.Perm(len(rec.rvolumes)) {
         remote = fmt.Sprintf("http://%s%s", rec.rvolumes[vn], key2path(key))
-        if remote_head(remote) {
+        if remote_head(remote, authtoken) {
```

Comment:
Hmm, can you include the basicauth in the remote url?

Reply:
You mean something like this `http://localhost:3001/sv07/60/08/L3dlaGF2ZQ==?authtoken=XXXXXXX`.

And then using a function to transform that into `req.Header.Set("Authorization", authtoken)`?

Reply:
Oh! you mean `http://admin:thisisatest@localhost:3001/sv07/60/08/L3dlaGF2ZQ==`